tests/models/test_renderedtifields.py

```diff
@@ -0,0 +1,217 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Unit tests for RenderedTaskInstanceFields."""
+
+import unittest
+from datetime import date, timedelta
+
+from parameterized import parameterized
+
+from airflow import settings
+from airflow.models import Variable
+from airflow.models.dag import DAG
+from airflow.models.renderedtifields import RenderedTaskInstanceFields as RTIF
+from airflow.models.taskinstance import TaskInstance as TI
+from airflow.operators.bash import BashOperator
+from airflow.utils.session import create_session
+from airflow.utils.timezone import datetime
+
+TEST_DAG = DAG("example_rendered_ti_field", schedule_interval=None)
+START_DATE = datetime(2018, 1, 1)
+EXECUTION_DATE = datetime(2019, 1, 1)
+
+
+def clear_rendered_ti_fields():
```

Comment:
Can you move this method to tests/utils/?

Reply:
Moved